frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx

```diff
@@ -28,16 +33,19 @@ export function SetupScannerScreen({ batteryIsCharging }: Props): JSX.Element {
           </Text>
         </CenteredLargeProse>
       )}
+      {scannedBallotCount && <ScannedBallotCount count={scannedBallotCount} />}
```

Comment:
What if the `scannedBalotCount` is `0`? We currently won't display a count in that case - is that what we want?

Reply:
🙏  Fixing…